brave/RATIONALE.md

```diff
@@ -1,5 +1,29 @@
 # brave rationale
 
+## SpanListener
```

Comment:
ignore drift here for now as I need to rewrite it anyway but don't want to rewrite it two or three times